webserver/views/api/v1/core.py

```diff
@@ -60,17 +60,21 @@ def get_high_level(mbid):
     for an MBID, you can browse through them by specifying an offset parameter
     ``n``. Documents are sorted by the submission time of their associated
     low-level documents.
+    You can get the full key values for model classes by specifying a
+    parameter ``map=True``
 
     You can get the total number of low-level submissions using ``/<mbid>/count``
     endpoint.
 
     :query n: *Optional.* Integer specifying an offset for a document.
+         map: *Optional.* Boolean flag to get mapped keys
```

Comment:
check the syntax of this query for generating docs. I think that it should be `:query map:` to correctly build the docs. you should build them and verify the output.